hapi-proto/src/main/proto/ScheduleCreate.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2020 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+option java_multiple_files = true;
+
+import "BasicTypes.proto";
+import "Timestamp.proto";
+
+message ScheduleCreateTransactionBody {
+  bytes transactionBody = 1; // The transaction serialized into bytes that must be signed
+  Key adminKey = 2; // (optional) The Key which is able to delete the Scheduled Transaction (if tx is not already executed)
+  AccountID payer = 3; // (optional) The account which is going to pay for the execution of the Scheduled TX. If not populated, the scheduling account is charged
+  SignatureMap sigMap = 4; // The signature map containing the signature of at least one account defined in the signers proto on transaction creation
+  bool executeImmediately = 5; // Whether to execute the transaction immediately when all required signatures are collected. Currently only executeImmediately=true is supported
```

Comment:
You are right about the fact that later we could think that it would be better to have enum or use another property or use `executeAt`. 
I do think that we can use 1 property for determining the execution time (immediately or at a given time). In the full version of scheduled transactions, we could add `executeAt` property (timestamp) and if its left empty we could deduce that it should be "executed immediately".

Lets discuss it on our weekly sync calls with the rest of the team.

Reply:
I'd go with YAGNI here and remove both `executeImmediately` and `executeAtTime` unless there is already groundwork laid in other branches for the ultimate use of these fields.

Reply:
I agree, as a maintainer a downstream project, having dead code out of the box complicates our efforts.  Perhaps leave a comment in the protobuf saying these features are under consideration.  For example, the confusion around staking and live hashes has needlessly consumed our time that could be better spent.

Reply:
We've updated the protobufs by removing the 2 properties